package.json

```diff
@@ -22,7 +22,7 @@
     "check-types": "tsc --noEmit --project tsconfig.json",
     "coverage-report": "nyc report --reporter=text-lcov | coveralls",
     "docs-build": "typedoc --out docs src/index.ts",
-    "eslint:check": "eslint . --max-warnings 0 --ext .js,.ts",
+    "eslint:check": "eslint . --fix --max-warnings 0 --ext .js,.ts",
```

Comment:
Don't think we want this here--should fail instead of fixing. Could add another `eslint:fix` if you want